tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/lg/inject.lg

```diff
@@ -13,4 +13,7 @@
 -user.tasks
 
 #GetMessage
-- This is an injected message
\ No newline at end of file
+- This is an injected message
+
+#UserDataConcat()
```

Comment:
can you make this test case or create new test case to be able to test both local and global case?